local.settings.json.example

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
can we use a .env file instead? (so we can add commments)

Reply:
`local.settings.json` is the default file used by azure function just for local usage but in production, these values have to be populated in Application settings. I think it is not possible to rename it!

Reply:
maybe we can add a `local.settings.json.example` file

Reply:
@lucacavallaro already added in PR!